testing/e2e/bazel.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Title        bazel.sh
+# Description  Bazel utilities for setting up a WORKSPACE scoped test environment
+#================================================================================
+
+function write_workspace_file() {
+  local file_name=$1
+  local workspace_name=$2
+
+  export WIX_BUILD_TOOLS_DIR=${TEST_TMPDIR}/wix_build_tools
```

Comment:
not sure you need to `export` it...  it may be good enough as local env var
Also - seems like this function assumes that the `wix_build_tools` is under `${TEST_TMPDIR}/wix_build_tools` 
Maybe a `local wix_build_tools_dir=$3` would be better?

Reply:
You are correct, the export is wrongly used in here.. I'll export the `wix_build_tools` path on the function which actually creates that folder and use the exported var in here.